src/main/java/mods/eln/gridnode/GridDescriptor.java

```diff
@@ -62,13 +62,17 @@ public void draw(float idealRenderingAngle) {
 
     @Override
     public void renderItem(ItemRenderType type, ItemStack item, Object... data) {
-        objItemScale(obj);
```

Comment:
I converted this whole thing to Kotlin in a separate PR I haven't finished yet...

Oh, I won't stop you. But this will be fun.